packages/menu/package.json

```diff
@@ -0,0 +1,20 @@
+{
+   "name": "@ifixit/menu",
+   "version": "0.0.0",
+   "main": "./index.tsx",
+   "types": "./index.tsx",
+   "license": "MIT",
+   "scripts": {
+      "build": "",
+      "clean": "",
+      "type-check": "tsc --pretty --noEmit"
+   },
+   "devDependencies": {
+      "@ifixit/tsconfig": "workspace:*",
+      "@types/react": "18.0.24",
+      "@types/react-dom": "18.0.8",
+      "react": "18.2.0",
+      "react-dom": "18.2.0",
+      "typescript": "4.8.4"
+   }
```

Comment:
I don't know if it makes much different in the end, but I would call react + react-dom "peer" dependencies. I'd look at several other packages in this repo to see what they do.

Reply:
These were actually not needed. Removed in [caf0fc1](https://github.com/iFixit/react-commerce/pull/1479/commits/caf0fc1ff162d21da4e83821afb14e9eb7a2f5f9).

Other packages do list react/react-dom as devDependencies, not peer dependencies.